webpack/scenes/AlternateContentSources/Create/Steps/AcsUrlPaths.js

```diff
@@ -71,6 +90,15 @@ const AcsUrlPaths = () => {
             onChange={checked => setVerifySSL(checked)}
           />
         </FormGroup>
+        {verifySSL && acsType === 'rhui' &&
+          <>
+            {__('Generate certificates for the chosen repositories')}
```

Comment:
We've been standardizing around adding periods, so I've added some suggestions.

```suggestion
            {__('Generate certificates for the chosen repositories.')}
```